pywbem/mof_compiler.py

```diff
@@ -2089,14 +2096,21 @@ def CreateClass(self, *args, **kwargs):
         cc = args[0] if args else kwargs['NewClass']
         if cc.superclass:
             try:
-                _ = self.GetClass(cc.superclass, LocalOnly=True,  # noqa: F841
-                                  IncludeQualifiers=False)
+                # Since this may cause recursive GetClass calls
+                # IncludeQualifiers = True to insure reference properties on
+                # instances with aliases get built correctly.
```

Comment:
We are in CreateClass. This does not drive recursive GetClass calls. There is one check for the direct superclass, and if that fails, the CreateClass fails. It is up to the MOF input to define the classes in the right order, unless I missed something.

Reply:
Change wording to additional GetClass. The purpose of the note is that we need LocalOnly and IncludeQualifiers to be set True for the additional operations to work correctly.